### PR TITLE
Cloud: fix code block in Snapshots topic

### PR DESCRIPTION
**Pull request by barnato-v**
Added ``` to command bash
php vendor/bin/ece-tools -- main sales to format correctly in the section titled Dump your database {#db-dump}

## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

Snapshots and backup management: https://devdocs.magento.com/cloud/project/project-webint-snap.html

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@barnato-v thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by dobooth** [comment]
Thanks @barnato-v for the contribution. Please sign the Adobe CLA so we can proceed with this PR.

File: src/cloud/project/project-webint-snap.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ By default, this command creates backups for all database connections that are s
 You can also backup only selected databases by appending the database names to the command, for example:
 
 ```bash
-php vendor/bin/ece-tools -- main sales
+php vendor/bin/ece-tools -- main sales ```
```
Comment (hguthrie) [inline src/cloud/project/project-webint-snap.md:125]: Thanks for catching that. Just need to place in on the next line and it's good.

**Review by hguthrie — CHANGES_REQUESTED** [review]

**Comment by barnato-v** [comment]
> @barnato-v thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

@hguthrie I tried clicking on the link you suggested but I didn't see an invitation? It redirected to https://github.com/orgs/magento

Can you resend?

**Review by hguthrie — APPROVED** [review]
Thanks for fixing!
I went ahead and removed the extra space and added the carriage return.
I'm not sure about that link you mentioned. I think that was an automated email. But if you want to fully participate, you can certainly ask in our Slack channel. 🙂 In any case, I will pass the word along.

**Comment by hguthrie** [comment]
running tests

**Comment by hguthrie** [comment]
running tests

**Pull request merged**